controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -236,7 +236,19 @@ http {
         ssl_verify_client on;
         ssl_verify_depth {{ $location.CertificateAuth.ValidationDepth }};
         {{ end }}
+        
+        {{ if (or $location.Redirect.ForceSSLRedirect (and (not (empty $server.SSLCertificate)) $location.Redirect.SSLRedirect)) }}
```

Comment:
Also @rikatz I'm not blocked here (I use a custom template anyway) - so enjoy your vacation :)